graylog2-web-interface/src/pages/ShowContentPackPage.jsx

```diff
@@ -47,16 +56,18 @@ const ShowContentPackPage = createReactClass({
     if (window.confirm('You are about to delete this content pack, are you sure?')) {
       ContentPacksActions.deleteRev(contentPackId, revision).then(() => {
         UserNotification.success('Content Pack deleted successfully.', 'Success');
-        ContentPacksActions.get(contentPackId);
+        ContentPacksActions.get(contentPackId).catch((error) => {
+          if (error.status !== 404) {
```

Comment:
Should we display a message when the error is a 404?

Reply:
No because we deleted the last content pack revision and that is okay, so we get to the content packs list silently.